config/webpack.config.js

```diff
@@ -93,40 +90,31 @@ const configGenerator = (options) => {
           use: {
             loader: 'babel-loader',
             options: {
-              presets: ['react'],
               // Speed up compilation.
               cacheDirectory: '.babelcache'
-
               // Also see .babelrc
             }
           }
         },
-        {
-          test: /foundation\.js$/,
-          use: {
-            loader: 'imports-loader?this=>window',
-          }
-        },
-        {
-          test: /modernizrrc/,
-          use: {
-            loader: 'modernizr-loader'
-          }
-        },
         {
           test: /\.scss$/,
           use: ExtractTextPlugin.extract({
             fallback: 'style-loader',
             use: [
               { loader: 'css-loader' },
               { loader: 'sass-loader' }
-            ],
+            ]
           })
         },
         {
+          // if we want to minify these images, we could add img-loader
+          // but it currently only would apply to three images from uswds
```

Comment:
There used to be an img-loader here, but it was never actually running. It doesn't seem necessary, given that it would only apply to 3 images if it did.